internal/ocm/resource/cluster.go

```diff
@@ -93,3 +98,79 @@ func (c *Cluster) CreateNodes(ctx context.Context, autoScalingEnabled bool, repl
 
 	return nil
 }
+
+func (c *Cluster) CreateAWSBuilder(awsTags map[string]string, ec2MetadataHttpTokens *string, kmsKeyARN *string,
+	isPrivateLink bool, awsAccountID *string, stsBuilder *cmv1.STSBuilder, awsSubnetIDs []string) error {
+
+	if isPrivateLink && awsSubnetIDs == nil {
+		return errors.New("Clusters with PrivateLink must have a pre-configured VPC. Make sure to specify the subnet ids.")
+	}
+
+	awsBuilder := cmv1.NewAWS()
+
+	awsBuilder.Tags(awsTags)
```

Comment:
I think it better to check if awsTags is not nil

Reply:
Done